crypto/pem_cert_pool.go

```diff
@@ -0,0 +1,70 @@
+package crypto
+
+import (
+	"encoding/pem"
+
+	"github.com/golang/glog"
+	"github.com/google/certificate-transparency/go/x509"
```

Comment:
Need to decide then as I've got several PRs that are pulling in this and the other CT specific packages.